extensions.html

```diff
@@ -334,6 +334,117 @@ <h2><dfn>GamepadPose</dfn> Interface</h2>
         </dd>
       </dl>
     </section>
+    
+    <section>
+      <h2><dfn>GamepadLightIndicatorType</dfn> Enum</h2>
+      <p>
+        This enum defines the type of light indicators supported by gamepads.
+      </p>
+      
+      <pre class="idl">
+        enum GamepadLightIndicatorType {
+          "on-off",
+          "rgb"
+        };
+      </pre>
+      
+      <dl data-dfn-for="GamepadLightIndicatorType">
+        <dt>"<dfn>on-off</dfn>"</dt>
+        <dd>
+          Light indicator that supports a single color.
+        </dd>
+        
+        <dt>"<dfn>rgb</dfn>"</dt>
+        <dd>
+          Light indicator that supports multiple colors.
+        </dd>
+      </dl>
+    </section>
+    
+    <section>
+      <h2><dfn>GamepadLightColor</dfn></h2>
+      <p>
+        This represents the color of a light indicator. The default value should
+        be set to 0 for red, green, and blue.
```

Comment:
Don't need the description, as default is already state in the IDL. 
```suggestion
        This represents the color of a light indicator.
```